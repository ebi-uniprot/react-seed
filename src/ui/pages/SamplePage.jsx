
import React, { Fragment } from 'react';

import SearchResultsPageLayout from '../layout/SearchResultsPageLayout';

const SamplePageContent = () => (
  <Fragment>
    <h3>Lorem ipsum dolor sit amet</h3>
    <p>
      Vivamus et sem ut nulla faucibus lobortis. Praesent in enim ultrices, pharetra
      nulla quis, pellentesque enim. Nunc vel accumsan sapien. Proin finibus vehicula
      tortor, ut scelerisque ligula egestas sed. In in vehicula lorem. Ut vel vestibulum
      nulla, non mattis purus. Praesent sit amet vestibulum orci, quis porttitor felis.
    </p>
  </Fragment>
);

const SamplePage = () => (
  <SearchResultsPageLayout
    content={<SamplePageContent />}
  />
);

export default SamplePage;
