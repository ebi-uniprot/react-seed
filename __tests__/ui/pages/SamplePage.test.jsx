
import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import SamplePage from '../../../src/ui/pages/SamplePage';

describe('SamplePage component', () => {
  test('should render', () => {
    const component = renderer
      .create(<MemoryRouter><SamplePage /></MemoryRouter>)
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
