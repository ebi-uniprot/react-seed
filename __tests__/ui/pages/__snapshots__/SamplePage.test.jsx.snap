// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SamplePage component should render 1`] = `
<div
  className="search-results-page-layout"
>
  <b>
    Search Results: 
  </b>
  <a
    href="/"
    onClick={[Function]}
  >
    Go back to Home page
  </a>
  <div
    className="default-page-conent"
  >
    <h3>
      Lorem ipsum dolor sit amet
    </h3>
    <p>
      Vivamus et sem ut nulla faucibus lobortis. Praesent in enim ultrices, pharetra nulla quis, pellentesque enim. Nunc vel accumsan sapien. Proin finibus vehicula tortor, ut scelerisque ligula egestas sed. In in vehicula lorem. Ut vel vestibulum nulla, non mattis purus. Praesent sit amet vestibulum orci, quis porttitor felis.
    </p>
  </div>
</div>
`;
